book/website/collaboration/inclusive-events/inclusive-events-comms.md

```diff
@@ -0,0 +1,487 @@
+Although you may be able to assume that many people with lived experience are familiar with any common acronyms or jargon used, 
+it can benefit the inclusion of all attendees and experience levels if the amount of jargon used can be minimized. 
+For example, researchers, clinicians and people with a lived experience of arthritis may be more familiar with terms and 
+medication names related to arthritis and associated conditions. 
+However, many attendees may be unfamiliar with these terms can can feel excluded from conversation and event content that focuses on them 
+
+As part of planning, consider how you can reduce the use of jargon and acronyms at your event, and create a strategy for supporting speakers and facilitators to also avoid unnecessary jargon. 
+
+You may want to develop a Glossary of Terms that can be included in information booklets, and where possible, work with researchers to be mindful of, 
+and review, language choices to ensure inclusivity, such as avoiding gender-specific terms and including language that recognizes the diverse identities engaged in your event. 
+
+You can find online examples of Glossaries developed by research funding bodies, or review the [NHS list of jargon and acronyms](https://www.nhsconfed.org/publications/acronym-buster) for inspiration.
+
+(cl-inclusive-events-comms-plain-language)=
+### Using Plain Language
+Similar to avoiding jargon, ensure that your core communications are written using Plain Language or are in an "easy reading" format. 
+
+Using plain language is especially important in your registration and event information packs.
+
+Plain language, sometimes referred to as Plain English in the UK and North America, allows for communications and other information to be read and understood easier. 
+Consider how you will encourage attendees and speakers to adopt clear and plain language communication throughout your event. 
+
+For example, you could address and encourage the use of plain language during the opening address or by signposting it in your event's shared information. 
+You could also consider an activity where researchers are paired with non-researchers and are asked to explain their research in under 3 minutes. 
+This can help to bring "clear and effective communication" to the forefront of attendee's minds and set all attendees up to communicate clearly throughout the event.
+
+If possible, consider co-developing plain language and Easy Read content alongside a community representative or person with lived experience. 
+If this is not possible for your event, ensure that core communications and event content are checked by using an external tool such as the [Hemingway App](https://hemingwayapp.com/). 
+You can also paste the URL of a website into a [Readability Checker](https://www.webfx.com/tools/read-able/) to check the content of your event's webpage. 
+There are more resources below that can help you prepare Plain Language content. 
```

Comment:
I think it would be good to either remove the resources list below or remove the concrete examples here and refer to the list below exclusively. 

As is there's a slight mismatch in that e..g Readability checker isn't listed below, but Hemingway is repeated. Plus there's only 2 more references below in the resource list that aren't already linked to here.

Reply:
Integrated the two paragraphs and added a call out block to help make the overall section more readable.